app/src/models/pull-request.ts

```diff
@@ -25,6 +25,12 @@ export class PullRequestRef {
   }
 }
 
+/** The commit status and metadata for a given ref */
+export interface ICombinedRefStatus {
```

Comment:
The "combined" nomenclature is unclear to me. It seems like this is the opposite of a combined ref status, since it's an individual status per check? Or maybe I'm not understanding what all is happening here?

Reply:
I think it just has the latest status for each check (which is what we want) as opposed to every status set on that commit for each check.

Reply:
Yea... that name is actually from the API [docs](https://developer.github.com/v3/repos/statuses/#get-the-combined-status-for-a-specific-ref). My understanding is it combines all the statuses from the integrations you have set up for your repository.

In our case, we'd have 3 `ICombinedRefStatus` objects(?). One for each of our CI integration. The `PullRequestStatus` object has a property for keeping track of all `ICombinedRefStatus` objects.

https://github.com/desktop/desktop/blob/6c573ace7cbc47e5d12459f66ca12bb1910dd185/app/src/models/pull-request.ts#L48

Reply:
Also, I'm open to naming if that name doesn't make sense.

Reply:
From the API:

>access a combined view of commit statuses for a given ref. [...] The most recent status for each context is returned

My reading is that the result of the API is the combined view for _individual_ commit statuses. Or in other words, the `statuses` field is combined, but the individual statuses are _not_ the combination of anything. They're just individual statuses.

Reply:
How about `commitStatus` or `refStatus`?